docs/content/integrations/airflow/reference.mdx

```diff
@@ -0,0 +1,266 @@
+    os.path.join(os.getenv("AIRFLOW_HOME"), "dags"),
+    "migrated_airflow_repo",
+)
+```
+
+---
+
+## Dagster job to Airflow DAG
+
+To convert a Dagster Job to a Airflow Dag[dagster repository](/concepts/repositories-workspaces/repositories), use <PyObject module="dagster_airflow" object="make_airflow_dag" />:
+
+```python file=/integrations/airflow/hello_cereal_dag.py
+# pylint: disable=unused-variable
+
+import datetime
+
+from dagster_airflow.factory import make_airflow_dag
+
+DEFAULT_ARGS = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime.datetime(2019, 11, 7),
+    "email": ["airflow@example.com"],
+    "email_on_failure": False,
+    "email_on_retry": False,
+}
+
+dag, tasks = make_airflow_dag(
+    module_name="docs_snippets.integrations.airflow.hello_cereal",
+    job_name="hello_cereal_job",
+    dag_kwargs={"default_args": DEFAULT_ARGS, "max_active_runs": 1},
+)
+```
+
+---
+
+## Orchestrating Dagster jobs from Airflow
+
+You can orchestrate Dagster job runs from Airflow by using the `DagsterCloudOperator` or `DagsterOperator` operators in your existing Airflow DAGs. For example, here's an Airflow DAG:
+
+```python file=/integrations/airflow/operator.py
+from datetime import datetime
+
+from airflow import DAG
+from dagster_airflow import DagsterCloudOperator
+
+with DAG(
+    dag_id="dagster_cloud",
+    start_date=datetime(2022, 5, 28),
+    schedule_interval="*/5 * * * *",
+    catchup=False,
+) as dag:
+    DagsterCloudOperator(
+        task_id="new_dagster_assets",
+        repostitory_location_name="example_location",
+        repository_name="my_dagster_project",
+        job_name="all_assets_job",
+    )
+```
+
+In Airflow 2.0+, you can create a Dagster connection type to store configuration related to your Dagster Cloud organization. If you're using Airflow 1.0, you can also pass this directly to the operator.
+
+---
+
+## Exporting Dagster jobs to Airflow
+
+You can compile Dagster jobs into DAGs that can be understood by Airflow. Each op in the job becomes an Airflow task. For example, here's a Dagster job:
+
+```python file=/integrations/airflow/hello_cereal.py
+import csv
+
+import requests
+
+from dagster import job, op
+
+
+@op
+def download_cereals():
+    response = requests.get("https://docs.dagster.io/assets/cereal.csv")
+    lines = response.text.split("\n")
+    return [row for row in csv.DictReader(lines)]
+
+
+@op
+def find_sugariest(context, cereals):
+    sorted_by_sugar = sorted(cereals, key=lambda cereal: cereal["sugars"])
+    context.log.info(f'{sorted_by_sugar[-1]["name"]} is the sugariest cereal')
+
+
+@job
+def hello_cereal_job():
+    find_sugariest(download_cereals())
+```
+
+To make this job available inside Airflow, you can write an Airflow DAG definition file that invokes <PyObject module="dagster_airflow" object="make_airflow_dag" />:
+
+```python file=/integrations/airflow/hello_cereal_dag.py
+# pylint: disable=unused-variable
```

Comment:
[1] I recommend adding `startbefore` and `endafter` markers to this example so the `pylint` line doesn't show up in the rendered example.